exporter/datasetexporter/logs_exporter.go

```diff
@@ -65,6 +66,73 @@ func buildBody(attrs map[string]interface{}, value pcommon.Value) string {
 	return message
 }
 
+// Function maps OTel severity on the LogRecord to DataSet severity level (number)
+func otelSeverityToDataSetSeverity(log plog.LogRecord) int {
```

Comment:
what about mapOtelSeverityToDataSetSeverity ?

Reply:
That's fine with me. 

I'm not an expert on go and otel conventions, but I do know they try to avoid "get" prefix in the function names so I left it out. I guess "map" may be fine.

Reply:
From their docs:

> Go doesn't provide automatic support for getters and setters. There's nothing wrong with providing getters and setters yourself, and it's often appropriate to do so, but it's neither idiomatic nor necessary to put Get into the getter's name. If you have a field called owner (lower case, unexported), the getter method should be called Owner (upper case, exported), not GetOwner. The use of upper-case names for export provides the hook to discriminate the field from the method. A setter function, if needed, will likely be called SetOwner. Both names read well in practice

And technically this method is a getter. But I'm also fine with adding `map` prefix (I do prefer prefix myself since this method is actually a bit more than a simple getter).

Reply:
Yes I agree this is not getter, i takes argument so lets stick with map

Reply:
Done.